sdk/src/beta9/abstractions/volume.py

```diff
@@ -1,29 +1,73 @@
+from dataclasses import dataclass
 from typing import Optional
 
 from ..abstractions.base import BaseAbstraction
-from ..clients.gateway import Volume as VolumeConfig
+from ..clients.gateway import MountPointConfig as VolumeConfigGateway
+from ..clients.gateway import Volume as VolumeGateway
 from ..clients.volume import GetOrCreateVolumeRequest, GetOrCreateVolumeResponse, VolumeServiceStub
 
 
+@dataclass
+class VolumeConfig:
+    """
+    Configuration for a volume.
+
+    Parameters:
+        external (bool):
+            Whether the volume is from an external provider.
+        access_key (str):
+            The S3 access key for the external provider.
+        secret_key (str):
+            The S3 secret key for the external provider.
+        endpoint (Optional[str]):
+            The S3 endpoint for the external provider.
+    """
+
+    external: bool = False
+    access_key: Optional[str] = None
+    secret_key: Optional[str] = None
+    endpoint: Optional[str] = None
+
+
 class Volume(BaseAbstraction):
```

Comment:
Can we create a CloudBucket(Volume) class that basically just does exactly what you've done here with the config? I'm not saying replace the work you've done, just make it into a new name from a client perspective so it's clearly delineated.